docs/languages/Csharp.md

```diff
@@ -15,6 +15,17 @@ There are special use-cases that each language supports; this document pertains
 
 Sometimes you want to serialize the data models into JSON. In order to do that use the preset `CSHARP_JSON_SERIALIZER_PRESET`
 
+**Important Note:**
+The following dependencies used in the serializer and deserializer functionality are .NET specific dependencies.
+
+```
+System.Text.Json
+System.Text.Json.Serialization
+System.Text.RegularExpressions
+```
```

Comment:
Now that we have `External dependencies` do you think we need `Important Note` as well? 🤔 They are kinda saying the same thing 🙂

Reply:
Yeah, corrected.